github/ci/testgrid/hack/common.sh

```diff
@@ -30,6 +30,7 @@ generate_config(){
 
 run_tests(){
     (
+        ${PROJECT_INFRA_ROOT}/hack/create_bazel_cache_rcs.sh
```

Comment:
Can't you just transparently enable the cache by simple setting the bazel preset label on the job?

Something like:

```
preset-bazel-cache: "true"
```

this way we would stay in sync with what we use as bootstrap image, and we don't lookup a non-existing cache if things are called locally.

Reply:
Good point, added the preset label to set the proper env vars. However we still need to call the `create_bazel_cache_rcs.sh` script to set up bazelrc settings, for the testgrid config management jobs we need to use the transfigure image `gcr.io/k8s-prow/transfigure` (which ships configurator and other required tools) instead of our bootstrap image (on which we execute `create_bazel_cache_rcs.sh` during image build).

Reply:
ah :+1: